tests/test_utils.py

```diff
@@ -303,3 +306,437 @@ def check_multistream_classic(fnames):
+
+        len_dates = len(snotel_swe['DATES'])
+
+        # plotting preferences
+        lw = 4.0
+        # loop over variables to plot
+
+        for i, plot_variable in enumerate(plot_variables):
+
+            if 'water_year' in plots_to_make:
+
+                plt.figure(figsize=(10,10))
+
+                if plot_variable == "OUT_SWE":
+
+                    # plot SnoTel SWE observations
+                    plt.plot(snotel_swe['DATES'], snotel_swe[plot_variable],
+                            'k', label='Snotel', linewidth=lw)
+
+                # plot VIC 4.2 simulations
+                plt.plot(snotel_swe['DATES'], vic_42[plot_variable][:len_dates],
+                        'b', label='VIC 4.2', linewidth=lw)
+
+                # plot VIC 5.0 simulations
+                plt.plot(snotel_swe['DATES'], vic_50[plot_variable][:len_dates],
+                        'r', label='VIC 5.0', linewidth=lw)
+
+                # plot VIC 5.0.x simulations
+                plt.plot(snotel_swe['DATES'], vic_50x[plot_variable][:len_dates],
+                        'y', label='VIC 5.0.x', linewidth=lw)
+
+                plt.title(plot_variable)
+                plt.legend(loc='upper left')
+                plt.ylabel(plot_units[i])
+
+                # save figure
+                os.makedirs(os.path.join(plot_dir, plot_variable), exist_ok=True)
+                plotname = '%s_%s.png' % (lat, lng)
+                savepath = os.path.join(plot_dir, plot_variable, plotname)
+                plt.savefig(savepath, bbox_inches='tight')
+
+                plt.clf()
+                plt.close()
+
+
+def plot_fluxnet_comparison(driver, testname, result_dir, plot_dir, vic_42_dir, vic_50_dir,
+                        obs_dir, plots_to_make):
+
+    ''' makes Ameriflux figures
+
+    '''
+
+    # loop over Ameriflux sites
+    for subdir in os.listdir(obs_dir):
+
+        lat_issue = False
+
+        if os.listdir(os.path.join(obs_dir, subdir)) != []:
+
+            # some sites do not have data
+            subdir_files = True
+
+            # get CSV file from site directory to get lat/lng for site
+            try:
+                site_csv_file = glob.glob(os.path.join(obs_dir, subdir, 'AMF*.csv'))[0]
+            except IndexError:
+                site_csv_file = glob.glob(os.path.join(obs_dir, subdir, 'us*.csv'))[0]
+            with open(site_csv_file) as file:
+                second_line = list(file)[1]
+
+            # parse line from header to get lat/lng
+            str_split = tsplit(second_line, ('Latitude: ', 'Longitude: ', 'Elevation (masl): '))
+            lat = str_split[1].strip()
+            lng = str_split[2].strip()
+
+            # load Ameriflux data
+            filename = '%s.stdfmt.hourly.local.txt' %subdir
+
+            # column names for DataFrame
+            # column names
+            names = ['Year', 'Month', 'Day', 'Hour', 'P', 'Tair', 'SWdown', 'LWdown', 'RH', 'Patm',
```

Comment:
rename this variable to ecflux names so it is clear that this is just for the obs.